test/benchmarks/README.md

```
@@ -17,6 +17,14 @@ To filter out which benchmarks are performed use:
 --benchmark_filter="adler32*"
 ```
 
+*Cooldown*
+
+To insert a cooldown sleep between benchmark runs to mitigate thermal throttling:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)